fails:Hash#update raises a RuntimeError if called on a frozen instance
